AUTHORS

```diff
@@ -827,3 +827,6 @@ Wassim Chegham <1699357+manekinekko@users.noreply.github.com>
 David Chin <dlcmhd@me.com>
 David Walker <d0sboots@gmail.com>
 Boris Verkhovskiy <boris.verk@gmail.com>
+JSKitty <jskitty@protonmail.com>
+CommanderRoot <CommanderRoot@users.noreply.github.com>
+npm cli ops bot <ops+npm-cli@npmjs.com>
```

Comment:
we need to add this email to the authors script ignore list

Reply:
OR we need to update this thing's name to include `[bot]` which appears to be the convention.